observer/baseObserverProvider.go

```diff
@@ -1,33 +0,0 @@
-package observer
```

Comment:
why you renamed this file because a full history node is an observer ?

Reply:
basically in order to make a difference between a simple observer and a full history node (even if it is also an observer)